src/goose/defaults.clj

```diff
@@ -12,6 +12,10 @@
 (def ^:no-doc process-prefix "goose/processes:")
 (def ^:no-doc heartbeat-prefix "goose/heartbeat:")
 
+(def ^:no-doc batch-prefix "goose/batch")
+(defn batch-successful-set [batch-id] (str d/prefix-batch "/" batch-id ":successful"))
+(defn batch-failed-set [batch-id] (str d/prefix-batch "/" batch-id ":failed"))
+(defn batch-enqueued-set [batch-id] (str d/prefix-batch "/" batch-id ":enqueued"))
```

Comment:
These can be within batch ns?
They are definitely not defaults

Reply:
Thats fair point. My understanding of putting it here was all the broker related names are put here, but since the batch related metadata is concerned for only batch, it makes sense to it put batch file.

Reply:
As decided from prev decision, all the name construct of keys lies in defaults namespace.